src/components/UI/Badge/Badge.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import classes from './Badge.module.css'
+
+const Badge = () => {
+    return (
+        <div className={classes.container}>
+            <div className={classes.badge}>
+                <p>
+                    Rank 1st{' '}
+                    <i className={`fas fa-trophy ${classes.trophy_icon}`}></i>

```

Comment:
Maybe we can change 
```javascript
<i className={`fas fa-trophy ${classes.trophy_icon}`}></i>
```
to this emoji 🏆.

Reply:
I thought that too. Just adding the trophy 🏆 icon works great too!